test/JDBC/input/BABEL-2819-vu-prepare.sql

```diff
@@ -0,0 +1,50 @@
+create view BABEL_2819_vu_prepare_v1 as (select datepart(dayofyear, cast('2016-11-14' as datetime)) as dayofyear, datepart(y, cast('2016-11-14' as datetime)) as y);
```

Comment:
have we validated that SQL server returns the same output?

Reply:
Just want to verify 'y' unit is the same as 'dayofyear' and they return the same value, cause they are not return the same value before.

Reply:
I assume SQL server has same expectations

Reply:
Yes, SQL server also returns the same as the BabelFish output.